### PR TITLE
#89-edit-api-readme

### PR DESCRIPTION
**Pull request by Fahi28**
Removed the region endpoint from the README since we no longer require that endpoint.

### CODE REVIEW

**Review by jiuliangut — DISMISSED** [review]
looks good! nice way to keep our documentation clear

File: api/README.md [hunk 1]
```diff
@@ -39,7 +31,7 @@
 - **Example**:
   GET `/earthquakes/date?start_date=01-01-2001&end_date=02-02-2002&sort=descending`
 
-### 5. Get Earthquakes by Alert Level
+### 4. Get Earthquakes by Alert Level
 - **Endpoint**: `GET /earthquakes/alert`
 - **Description**: Get earthquakes fo a specific alert level
 - **Query Parameters**:
```
Comment (Jakub-Poskrop) [inline api/README.md:37]: Looks good but I think the API endpoint for this is `GET /earthquakes/alert/<alert_level_colour: str> `

File: api/README.md [hunk 2]
```diff
@@ -20,30 +30,22 @@
...
+  - `end_date` (string, required): End date (YYYY-MM-DD)
   - `sort` (string, optional): Sort order for results:
     - `ascending`, `descending`; default is `ascending`
 - **Example**:
   GET `/earthquakes/date?start_date=01-01-2001&end_date=02-02-2002&sort=descending`
 
-### 5. Get Earthquakes by Alert Level
-- **Endpoint**: `GET /earthquakes/alert`
+### 4. Get Earthquakes by Alert Level
+- **Endpoint**: `GET /earthquakes/color`
```
Comment (Jakub-Poskrop) [inline api/README.md:45]: Shouldn't it be `/earthquakes/alert/colour`?

**Comment by S1mpySloth** [comment]
This seems to be somewhat behind your current iteration of the API README, should we maybe consider closing the pull request?

**Pull request closed without merging**